packages/lab/src/Flow/stories/Visualizations/Filter.tsx

```diff
@@ -8,19 +8,17 @@ import {
 import {
   HvFlowNode,
   HvFlowNodeFC,
-  HvFlowNodeTypeMeta,
   useFlowInputNodes,
   useFlowNodeUtils,
 } from "@hitachivantara/uikit-react-lab";
 
-import type { NodeGroup } from ".";
 import { NodeData } from "./data";
 
 function filterDataByCountries(data, countriesToFilter: string[]) {
   return data.filter((item) => countriesToFilter.includes(item.country));
 }
 
-export const Filter: HvFlowNodeFC<NodeGroup> = (props) => {
+export const Filter: HvFlowNodeFC = (props) => {
```

Comment:
We need to update line 35 for this node by spreading the previous data because when we check an option, the node title disappears.